README.md

```diff
@@ -171,6 +171,7 @@ then Lodash/Underscore is the better option.*
 1. [_.replace](#_replace)
 1. [_.split](#_split)
 1. [_.startsWith](#startsWith)
+1. [_.endsWith](#endsWith)
```

Comment:
could you resolve the alphatic order?  ( This item can be placed on top of _.padStart )

Reply:
just changed the order!